sktime/transformations/multiplexer.py

```diff
@@ -0,0 +1,253 @@
+        self,
+        transformers: list,
+        selected_transformer=None,
+    ):
+        super(MultiplexTransformer, self).__init__()
+        self.selected_transformer = selected_transformer
+
+        self.transformers = transformers
+        self.transformers_ = self._check_estimators(
+            transformers,
+            attr_name="transformers",
+            cls_type=BaseTransformer,
+            clone_ests=False,
+        )
+        self._set_transformer()
+        self.clone_tags(self.transformer_)
+        self.set_tags(**{"fit_is_empty": False})
+
+    def _check_selected_transformer(self):
+        component_names = self._get_estimator_names(
+            self.transformers_, make_unique=True
+        )
+        selected = self.selected_transformer
+        if selected is not None and selected not in component_names:
+            raise Exception(
+                f"Invalid selected_forecaster parameter value provided, "
+                f" found: {selected}. Must be one of these"
+                f" valid selected_forecaster parameter values: {component_names}."
+            )
+
+    def _set_transformer(self):
+        self._check_selected_transformer()
+        # clone the selected transformer to self.transformer_
+        if self.selected_transformer is not None:
+            for name, transformer in self._get_estimator_tuples(self.transformers):
+                if self.selected_transformer == name:
+                    self.transformer_ = clone(transformer)
+        else:
+            # if None, simply clone the first forecaster to self.forecaster_
+            self.transformer_ = clone(self._get_estimator_list(self.transformers)[0])
+
+    def _fit(self, y, X=None):
+        """Fit forecaster to training data.
+
+        private _fit containing the core logic, called from fit
+
+        Writes to self:
+            Sets fitted model attributes ending in "_".
+
+        Parameters
+        ----------
+        y : guaranteed to be of a type in self.get_tag("y_inner_mtype")
+            Time series to which to fit the forecaster.
+            if self.get_tag("scitype:y")=="univariate":
+                guaranteed to have a single column/variable
+            if self.get_tag("scitype:y")=="multivariate":
+                guaranteed to have 2 or more columns
+            if self.get_tag("scitype:y")=="both": no restrictions apply
+        fh : guaranteed to be ForecastingHorizon or None, optional (default=None)
+            The forecasting horizon with the steps ahead to to predict.
+            Required (non-optional) here if self.get_tag("requires-fh-in-fit")==True
+            Otherwise, if not passed in _fit, guaranteed to be passed in _predict
+        X : optional (default=None)
+            guaranteed to be of a type in self.get_tag("X_inner_mtype")
+            Exogeneous time series to fit to.
+
+        Returns
+        -------
+        self : reference to self
+        """
+        self._set_transformer()
```

Comment:
`__init__` is actually run now at the start every time `fit` is run, so this could be *completely empty* imo

Reply:
i.e., I think it should be ok to remove `_fit`

Reply:
Ah yea, I did forget about that change.  Sounds good.